skyportal/services/test_server/test_api_server.py

```diff
@@ -124,6 +124,33 @@ def submit_type(uri):
     assert r1_type == r2_type and r1.method == r2.method
 
 
+def ztf_request_matcher(r1, r2):
+    """
+    Helper function to help determine if two requests to the ZTF API are equivalent
+    """
+
+    # A request matches an LCO request if the URI and method matches
+
+    r1_uri = r1.uri.replace(":443", "")
+    r2_uri = r2.uri.replace(":443", "")
+
+    def submit_type(uri):
+        patterns = {
+            "delete": r"/api/triggers/ztf",
+            "put": r"/api/triggers/ztf/",
+        }
+        for (submit_type, pattern) in patterns.items():
+            if re.search(pattern, uri) is not None:
+                return submit_type
+
+        return None
+
+    r1_type = submit_type(r1_uri)
+    r2_type = submit_type(r2_uri)
```

Comment:
Since the routes for the DELETE and PUT are the same, I don't think we need to do all this submit type/pattern stuff. I'd refactor this to just be like a `is_ztf_request` function which just searches for the `/api/triggers/ztf` pattern and returns the boolean without the `submit_type` stuff. That, plus the `r1.method == r2.method` check should be enough.

Reply:
@kmshin1397 great point, see update to see if i did what you were imagining.